pkg/unidling/controller/unidling_controller.go

```diff
@@ -108,8 +108,11 @@ func NewUnidlingController(scaleNS scale.ScalesGetter, mapper meta.RESTMapper, e
 				}
 				p := pager.New(pager.ListPageFunc(fn))
 
-				options.FieldSelector = fieldSelector.String()
-				options.Limit = 500
+				// FIXME: that's a low limit to increase the api-server list requests
```

Comment:
"to increase"? what does it mean?

Reply:
@sttts means a temporary change to make sure it will trigger "chunking". I'll clean this up.

Reply:
Just pushed the changes in a single commit.